src/support/kuzzle.ts

```diff
@@ -4,7 +4,7 @@ import chalk from 'chalk'
 // tslint:disable-next-line
 const { Http, WebSocket, Kuzzle } = require('kuzzle-sdk')
 
-const SECOND = 60 * 1000
+const SECOND = 1000
```

Comment:
I guess I did not pay attention to this either when I reviewed this feature :expressionless:

Reply:
:sweat_smile: